Exam/CC1/src/exercice4/ReservationManager.java

```diff
@@ -0,0 +1,52 @@
+package exercice4;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Hashtable;
+import java.util.Set;
+
+public class ReservationManager {
+	private final Hashtable<Integer, Reservation> reservations = new Hashtable<Integer, Reservation>();
+	
+	public Reservation getReservation(Integer num) {
+		return reservations.get(num);
+	}
+	public Reservation getReservationDate(String date) throws ParseException {
```

Comment:
Several `Reservation` could correspond to a `Date`
Prototype should be something like:
~~~java
public List<Reservation> getReservationDate(String date)
~~~